src/org/robockets/stronghold/robot/drivetrain/Joyride.java

```diff
@@ -47,7 +47,10 @@ protected void execute() {
     		//Robot.driveTrain.encodersPID.reset();
     		//Robot.driveTrain.driveArcade(0, 0);
     	//} else {	
+		
+		if (!OI.buttonBoard1.getRawButton(6)) { // Don't allow Jack to turn the robot while trying to shoot
```

Comment:
can there be a global boolean somewhere that is "preventManualDriving"? Then a central place that sets this condition continuously. It should also prevent joystick input, so future vision code can drive the robot to get better alignment.